src/components/PeopleManagement/CreateGroupModal.jsx

```diff
@@ -24,6 +27,8 @@ const CreateGroupModal = ({
   const [groupName, setGroupName] = useState('');
   const [canCreateGroup, setCanCreateGroup] = useState(false);
   const [canInviteMembers, setCanInviteMembers] = useState(false);
+  const [isCreateGroupFileUpload, setIsCreateGroupFileUpload] = useState(false);
+  const [isCreateGroupListSelection, setIsCreateGroupListSelection] = useState(false);
```

Comment:
These should be reset to false somewhere after segment events are sent, in the event admins create more than one group back to back.